libraries/tlv-account-resolution/src/state.rs

```diff
@@ -232,7 +191,7 @@ mod tests {
                 .iter()
                 .map(ExtraAccountMeta::from)
                 .collect::<Vec<_>>(),
-            metas.iter().map(ExtraAccountMeta::from).collect::<Vec<_>>()
+            metas.to_vec()
```

Comment:
micro-nit: is the `to_vec()` needed? it should be able to do the equality, maybe by slicing?

Reply:
Weird I swore it barfed on me hah! Dropped